docs/components/receive.md

```diff
@@ -30,6 +30,15 @@ thanos receive \
     --objstore.config-file "bucket.yml"
 ```
 
+The example of remote_write Prometheus configuration:
+
+```yaml
+remote_write:
+- url: http://<thanos-receive-container-ip>:10908/api/v1/receive
+```
+
+where `<thanos-receive-containter-ip>` is an IP address reachable by Prometheus Server. In case of K8s sidecar container it could be just `0.0.0.0`.
```

Comment:
```suggestion
where `<thanos-receive-containter-ip>` is an IP address reachable by Prometheus Server.
```

This depends on the setup and if they run on the same network namespace, let's not environment-specific details.